qualang_tools/voltage_gates/voltage_gate_sequence.py

```diff
@@ -335,7 +335,8 @@ def add_compensation_pulse(self, max_amplitude: float = 0.49, **kwargs) -> None:
                     amplitude = declare(fixed)
                     eval_average_power = declare(int)
                     assign(eval_average_power, self.average_power[i])
-                    assign(amplitude, -Cast.mul_fixed_by_int(0.0009765625 / duration, eval_average_power))
+                    assign(amplitude, -Cast.mul_fixed_by_int(0.01 / duration, eval_average_power))
+                    assign(amplitude, amplitude * 0.09765625)
```

Comment:
This looks like a magic number, can you write the calculation for it?
Either directly or in a comment.

Reply:
It is not really magic, it corresponds to `1/1024` to compensate for the multiplication of the average voltage by 1024 to avoid digitization issues as it must be an int.
I am adding it as a comment